### ISSUE TITLE
Immersive Citizens + Dawnstar Arthmoor [flagged incompatable]

### ISSUE THREAD

**Issue by easelgames** [loot/loot#1179]
Latest LOOT version Version 0.15.1 (build 82c756e1)

![LOOT_hLL0efdJEY](https://user-images.githubusercontent.com/46850157/73183969-ad767280-40e9-11ea-90e1-664486194e22.png)

This is contrary to the mod page for Dawnstar by Arthmoor who says to simply load IC BEFORE.

![chrome_iCvh7stJ2F](https://user-images.githubusercontent.com/46850157/73184049-cc750480-40e9-11ea-9af4-93fb2ec71b0e.png)

I assume this is the case for all his other mod expansions to vanilla citys no? If this is incorrect however, please close.
Dawnstar by Arthmoor: https://www.nexusmods.com/skyrimspecialedition/mods/13607

**Comment by easelgames** [comment]
shor's stone
ivarstead
kynesgrove
soljund's sinkhole
karthwasten
dragon bridge

These are the files I manually needed to add this "load AFTER immersive citizens" rule. Their may be others.. but I checked all the ones I had.

**Comment by MacSplody** [comment]
> Rechecking things, it seems loot is NOT putting his mods AFTER Immersive Citizens.. Most of them are being placed before IC.

Only mods with direct conflicts with `Immersive Citizens - AI Overhaul.esp` have been placed after it.

- `Provincial Courier Service.esp`.
- `Darkwater Crossing.esp`.
- `Dawnstar.esp`, even though it's incompatible.
- `Falkreath.esp`.
- `Fall of Granite Hill.esp`.
- `Rorikstead.esp`.
- `Whistling Mine.esp`.

> shor's stone
> ivarstead
> kynesgrove
> soljund's sinkhole
> karthwasten
> dragon bridge
> 
> These are the files I manually needed to add this "load AFTER immersive citizens" rule. Their may be others.. but I checked all the ones I had.

If someone tests and finds that the order has any effect on the rest, then their order can be reviewed.

---

> I assume this is the case for all his other mod expansions to vanilla citys no?

Compatibility notes for Immersive Citizens are outlined on http://nextgen-ai.com/immersive-citizens-se-compatibility-support/. 

If users want to ignore the compatibilty notes by Arnaud dOrchymont, they can and an incompatibility message isn't going to stop them.

**Comment by easelgames** [comment]
Is there a reason why Dawnstar in particular is considered "not compatible" even though the mod itself has IC placement mentioned? I realize this is not loot specific, but if it's simply incompatible with IC, why even have IC load order mentioned on Dawnstar.esp?

**Comment by MacSplody** [comment]
> Is there a reason why Dawnstar in particular is considered "not compatible" even though the mod itself has IC placement mentioned? I realize this is not loot specific, but if it's simply incompatible with IC, why even have IC load order mentioned on Dawnstar.esp?

I've tested them myself, and there are issues regardless of order. Teleporting characters, characters stuck in certain spots, it's pretty easy to test with `COC DawnstarExterior01` from the main menu.

> why even have IC load order mentioned on Dawnstar.esp?

Because people still use the two together despite the compatibility issues.

**Comment by easelgames** [comment]
So removing Dawnstar.esp will be enough? Or no? *not existing save*

**Comment by MacSplody** [comment]
> So removing Dawnstar.esp will be enough? Or no? _not existing save_

If you're in the middle of a playthrough I'd leave it. It's going to do more harm if you remove it.

**Comment by easelgames** [comment]
Nah, getting ready to start one.. removed just dawnstar.esp, and rebuilding patch RN

**Comment by jmarceno** [comment]
Arthmoor, believes that it is compatible (he has kind of a strong opnion on this apparently :P )-> https://www.reddit.com/r/skyrimmods/comments/9y2wu5/immersive_citizens_ai_overhaul_dawn_of_skyrim/

After seen this message of his, I've tried and have been playing with this combination for a year now, never found anything game breaking or had anything that I had to resort to console commands to fix in Danwstar.

Is there a message standard for "Minor inconsistances or incompatibilties"? I find the current message a bit misleading, as it suggests game breaking problemns.
